File: 448-find-all-numbers-disappeared-in-an-array/find-all-numbers-disappeared-in-an-array.java
class Solution {
    public List<Integer> findDisappearedNumbers(int[] nums) {
        int n = nums.length;

        int[] v = new int[n + 1];

        for (int i = 0; i < n; i++) {
            v[nums[i]] = 1;
        }

        List<Integer> ans = new ArrayList<>();

        for (int i = 1; i <= n; i++) {
            if (v[i] == 1) continue;
            else ans.add(i);
        }

        return ans;
    }
}

File: 448-find-all-numbers-disappeared-in-an-array/find-all-numbers-disappeared-in-an-array.cpp
class Solution {
public:
    vector<int> findDisappearedNumbers(vector<int>& nums) {
        ios_base::sync_with_stdio(false);
        int n = nums.size();

        vector<int> v(n+1, 0);

        for(int i = 0; i < n; i++){
            v[nums[i]] = 1;
        }

        vector<int> ans;

        for(int i = 1; i <= n; i++){
            if(v[i] == 1) continue;
            else ans.push_back(i);
        }

        return ans;
    }
};